onediff_comfy_nodes/__init__.py

```diff
@@ -46,6 +48,7 @@
     "ControlNetGraphSaver": "ControlNet Graph Saver",
     "SVDSpeedup": "SVD Speedup",
     "ModuleDeepCacheSpeedup": "Model DeepCache Speedup",
+    "OneDiffCheckpointLoaderSimple": "OneDiff Load Checkpoint",
```

Comment:
```suggestion
    "OneDiffCheckpointLoaderSimple": "Load Checkpoint - OneDiff",
```

插件显示名字改成这样？显示样式上就和原生的先对齐了，用户知道是同一个功能的改进版本